OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -62,14 +62,15 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			var tables = utility.ModData.ObjectCreator.GetTypesImplementing<ScriptGlobal>()
 				.OrderBy(t => t.Name);
 
-			Console.WriteLine("<h3>Global Tables</h3>");
+			Console.WriteLine("## Global Tables");
```

Comment:
The current version of docs.openra.net uses `###`.

Reply:
`h3` translates to `###` so I assume this is an error and not on purpose. Can't remember as this patch is so old.